lookup-create-app-version.js

```diff
@@ -30,6 +20,34 @@ async function getCreateAppVersion(inputVersion) {
       })
       .on('error', reject);
   });
+}
+
+async function getCreateAppVersion(inputVersion) {
+  // This is the old flow, where versions up to 0.4.x are considered to be direct create-app versions
+  const [major, minor] = inputVersion.split('.');
+  if (major === '0' && ['1', '2', '3', '4'].includes(minor)) {
+    return inputVersion;
+  }
+
+  // The new flow is to go look up the release manifest to grab the create-app version
+  // In case the manifest isn't available yet, we'll retry for a while
+  let manifest;
+  for (let attempt = 1; attempt <= FETCH_ATTEMPT_COUNT; attempt++) {
+    try {
+      manifest = await fetchManifest(inputVersion);
+      break;
+    } catch (error) {
+      console.error(
```

Comment:
maybe just warn or even info, and saying "retrying" or whatnot? if this is kinda common

Reply:
cba xP